### DISCUSSION TITLE
🧪 Ribbon controls

### DISCUSSION THREAD

**Discussion by 0x5bfa** [CommunityToolkit/Labs-Windows#500] [Experiments]
# Ribbon controls

## Problem Statement

_Add here_

### Default mode

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/745c99f2-f2ba-4109-b012-5e85a062f12b)

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/1452f5a2-1276-4150-86ba-da0b482a045c)

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/269a55d7-ecdd-4b2c-9499-d57e7df3893d)

### Compact mode

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/49c9ac96-a5df-4878-925d-bed1328c6a30)

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/c796a0cf-058b-48f2-9d68-80f50f0fa6ab)

## Proposed Solution

```xaml
Add here
```

### Ribbon

_Add here_

### RibbonItemSeparator

_Add here_

### RibbonItem

_Add here_

## Open Questions

_Add here_

## API Reference Docs

_Add here_

## References

_Add here_

**Comment by Sanlorng** [comment]
see [Porting TabbedCommandBar #243](https://github.com/CommunityToolkit/Windows/pull/243)

**Reply by 0x5bfa** [reply]
I'm not suggesting about Tabbing view. It's tather more inner controls.

**Comment by Arlodotexe** [comment]
This was added in https://github.com/CommunityToolkit/Labs-Windows/pull/546
